integration-libs/cdc/core/public_api.ts

```diff
@@ -1 +1,2 @@
-export * from './index';
+export * from './cdc-core.module';
+export * from './models/cms.model';
```

Comment:
Should index.ts be deleted like elsewhere?

Reply:
should deleted.